lib/modules/manager/mix/extract.ts

```diff
@@ -28,26 +30,24 @@ export async function extractPackageFile(
       } while (!contentArr[lineNumber].includes('end'));
       let depMatchGroups = depMatchRegExp.exec(depBuffer)?.groups;
       while (depMatchGroups) {
-        const { depName, datasource, currentValue, organization } =
-          depMatchGroups;
+        const { app, requirement, opts } = depMatchGroups;
+
+        const github = githubRegexp.exec(opts)?.groups?.value;
+        const organization = organizationRegexp.exec(opts)?.groups?.value;
+
+        const packageName = organization ? `${app}:${organization}` : app;
+        const currentValue = requirement || github;
```

Comment:
don't we need to set the github value to packageName? it seems its the repo name not the current version.

Reply:
When the `datasource` is set to `github`, we're also setting `skipReason` as we can't renovate packages fetched from a GitHub repo - they don't specify version requirements, and Hex basically fetches them and saves the commit hash in the lock file. So it doesn't really matter what we set the `packageName` to - as I understand from [this discussion](https://github.com/renovatebot/renovate/discussions/14885), it's the actual package name to be used later in the process (which we're skipping) and if in doubt we can just use `depName` which is what happens here.

Reply:
set it as depName and leave currentValue undefined